Tests/LeafKitTests/TestHelpers.swift

```diff
@@ -0,0 +1,267 @@
+import XCTest
+import NIOConcurrencyHelpers
+@testable import LeafKit
+
+/// Assorted multi-purpose helper pieces for LeafKit tests
+
+// MARK: - Helper Functions
```

Comment:
Pure string testing functions of Lexer, Parser, Render (without using an full LeafRenderer)